WordPress/src/main/java/org/wordpress/android/ui/prefs/ProfileInputDialogFragment.java

```diff
@@ -0,0 +1,97 @@
+package org.wordpress.android.ui.prefs;
+
+import android.app.AlertDialog;
+import android.app.Dialog;
+import android.app.DialogFragment;
+import android.content.DialogInterface;
+import android.os.Bundle;
+import android.text.TextUtils;
+import android.view.LayoutInflater;
+import android.view.View;
+
+import org.wordpress.android.R;
+import org.wordpress.android.util.AppLog;
+import org.wordpress.android.widgets.OpenSansEditText;
+import org.wordpress.android.widgets.WPTextView;
+
+public class ProfileInputDialogFragment extends DialogFragment {
+
+    private static final String TITLE_TAG = "TITLE";
+    private static final String INITIAL_TEXT_TAG = "INITIAL-TEXT";
```

Comment:
Nitpicking here: while it doesn't make a difference functionally, and just to keep consistency with other parts of the app - it would be great if the contents of static strings used to hold state are written in lower case and use underscore for spearating words, i.e. `INITIAL_TEXT_TAG = initial_text`